R/sig_table.R

```diff
@@ -0,0 +1,69 @@
+#' Generate a signature table for a validation report
+#'
+#' @return A dataframe created from the validation config containing a row for
+#'   each user with the columns: role, name_and_title, signature, and date.
+#'
+#' @export
+#'
+#' @importFrom purrr map_dfr
+vt_generate_sig_table <- function(){
+
+  # Pull all the usernames into a data.frame
+  people <- map_dfr(read_validation_config()$usernames, data.frame)
```

Comment:
dplyr/purrr is a rather large dependency to use this function. `do.call('rbind', lapply(read_validation_config()$usernames, data.frame))` achieves the same goal

Reply:
Thanks @thebioengineer I was just thinking the same